src/tests/ftest/container/snapshot_functionality_test.yaml

```diff
@@ -0,0 +1,31 @@
+# change host names to your reserved nodes, the
+# required quantity is indicated by the placeholders
+hosts:
+    test_machines:
+        - boro-A
+        - boro-B
+timeout: 100
+faults: !mux
```

Comment:
Don't think you need mux here. Can you please explain if you have it for any reason?

Reply:
The run-time is about 250-seconds.  Have updated with 500.